### ISSUE TITLE
Processing stuck at 95%

### ISSUE THREAD

**Issue by drax18** [avocode/avocode#692]
IMPORTANT: Please try to describe how did it happen.
I just wanted to import my psd file, and after that, the processing stucked at 95% and nothing happens
-----
App v.2.5.2
Source UID: 146839
```
Error: invalid distance code
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.

**Comment by hiliz007** [comment]
I downloaded avocode today and I have this problem with a few files that are maybe quite heavy
